integrations/docker/ci-only-images/chip-cirque-device-base/Dockerfile

```diff
@@ -15,6 +15,7 @@ WORKDIR /app
 
 RUN apt-get update \
   && apt-get install --no-install-recommends -y sudo git ca-certificates psmisc dhcpcd5 wpasupplicant wireless-tools \
+                                                gdb python3 python3-pip libcairo2-dev libjpeg-dev libgif-dev python3-dev \
```

Comment:
You can add avahi-utils package if this image will be used to simulate the controller.
This package might help with verifying if the discoverable node service is propagated using `avahi-browse -rt _chipc._udp` command.

Reply:
Added, thanks.